src/iop/rgblevels.c

```diff
@@ -116,6 +116,15 @@ int default_colorspace(dt_iop_module_t *self, dt_dev_pixelpipe_t *pipe, dt_dev_p
   return iop_cs_rgb;
 }
 
+const char *description(struct dt_iop_module_t *self)
+{
+  return dt_iop_set_description(self, _("adjust black, white and mid-gray points in RGB color space."),
+                                      _("corrective and creative"),
+                                      _("linear, RGB, scene-referred"),
```

Comment:
input and output are display-referred because the GUI is 0-100%, output and processing are non-linear,